src/community/blocks.tsx

```diff
@@ -0,0 +1,347 @@
+                                alignItems: 'center',
+                            })}
+                        >
+                            {title && (
+                                <div style={{paddingRight: '32px'}}>
+                                    <Text2 regular as="div">
+                                        {title}
+                                    </Text2>
+                                </div>
+                            )}
+                            <div>
+                                <Text2 regular color={vars.colors.textSecondary}>
+                                    {description}
+                                </Text2>
+                            </div>
+                        </div>
+                    )}
+                </>
+            )}
+        </div>
+    );
+};
+
+interface SimpleBlockProps {
+    image?: RendersNullableElement<typeof Image>;
+    description?: string;
+    'aria-label'?: string;
+}
+
+export const SimpleBlock: React.FC<SimpleBlockProps> = ({image, description, 'aria-label': ariaLabel}) => {
+    return (
+        <div aria-label={ariaLabel}>
+            <div
+                className={sprinkles({
+                    display: 'flex',
+                    alignItems: 'center',
+                })}
+            >
+                {image && (
+                    <div
+                        className={sprinkles({
+                            paddingRight: 16,
+                        })}
+                        style={{zIndex: '0'}}
+                    >
+                        {image}
+                    </div>
+                )}
+
+                <Text2 regular color={vars.colors.textSecondary}>
+                    {description}
+                </Text2>
+            </div>
+        </div>
+    );
+};
+
+interface InformationBlockProps {
+    title?: string;
+    description?: string | Array<string>;
+    value: string;
+    secondaryValue?: string;
+    valueColor?: string;
+    'aria-label'?: string;
+}
+
+export const InformationBlock: React.FC<InformationBlockProps> = ({
+    title,
+    description,
+    secondaryValue,
+    value,
+    valueColor = vars.colors.textPrimary,
+    'aria-label': ariaLabel,
+}) => {
+    return (
+        <Inline space="between" alignItems="flex-end" aria-label={ariaLabel}>
+            <BlockContent title={title} description={description} />
+            <div className={classNames(styles.column, styles.container)}>
+                <Text2 regular color={vars.colors.textSecondary}>
+                    {secondaryValue}
+                </Text2>
+                <Text5 color={valueColor}>{value}</Text5>
+            </div>
+        </Inline>
+    );
+};
+
+interface HighlightedValueBlockProps {
+    tag?: RendersNullableElement<typeof Tag>;
+
+    mainHeading: {
+        value: string;
+        text: string;
+    };
+
+    secondHeading?: {
+        value: string;
+        text: string;
+    };
+
+    secondaryValue?: string;
```

Comment:
In this case the secondHeading is a second header that is mandatory to have both value and text.

The secondaryValue, on the other hand, would be a second value (in relation to the mainHeading value) with a different color and without the need for a text next to it.

Before, an array of heading elements was allowed, but as suggested in this PR and because it only has 2 headings, with no further possibility, in the documentation, these two mainHeading and secondaryHeading objects were created

Reply:
hmmm.. I still see this quite confusing. Can I have secondHeading and secondaryValue at the same time in the component?

Reply:
yes, you can:
<img width="598" alt="image" src="https://github.com/Telefonica/mistica-web/assets/123959276/f2600374-71e9-490a-831a-81a21483b45b">